src/js/control-bar/volume-menu-button.js

```diff
@@ -65,6 +67,12 @@ class VolumeMenuButton extends MenuButton {
     this.on(this.volumeBar, ['sliderinactive', 'blur'], function(){
       this.removeClass('vjs-slider-active');
     });
+
+    // we show the volumeBar on focus for keyboard accessibility
+    // when using the mouse, you don't want it to stay up, so, we blur it manually
+    this.on('mouseout', function() {
+      this.el().blur();
```

Comment:
I think that may be ok. Or at least, something that we would need to live with.
Unless we want to (if we are even able to) know whether we entered into this via a keyboard or mouse and then only blur on mouseout if we entered with the mouse.